infrastructure/Jenkinsfile-CI.groovy

```diff
@@ -28,6 +28,13 @@ pipeline {
                 '''
             }
         }
+        stage('Build Keycloak extensions') {
+            steps {
+                sh'''
+                    mvn -B -U clean install
```

Comment:
We try to prevent executing maven directly on the server, so that we don't have to install different dependencies for different languages/tools. Could we run it inside a Docker container?